test/serializer/abstract/Throw8.js

```diff
@@ -26,7 +26,7 @@ var x = 1;
 try {
   call(fn);
   x = 2;
-  call(fn);
+  //call(fn);
```

Comment:
A comment of why this is getting (temporarily) disabled with the existing bug number would be nice.

Reply:
This can be obtained by blaming the line and referring back to this pull request.